src/app/ui/leaderboard/LeaderboardPage.tsx

```diff
@@ -0,0 +1,11 @@
+'use client';
+import { LeaderboardPage as LeaderboardPageComponent } from '@/components/Leaderboard/LeaderboardPage';
+
+const LeaderboardPage = ({ page }: { page?: string }) => {
+  const parsedPage = page ? parseInt(page, 10) : NaN;
```

Comment:
parseInt(page, 10) will have the same result without the if statement

Reply:
tried something not better, let me know what you think?